app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cash = {}
+
+    def decorator(*args, **kwargs) -> int | list:
```

Comment:
The return type hint `int | list` is not compatible with Python versions below 3.10. If compatibility with older versions is required, consider using `Union[int, list]` from the `typing` module.